mep/people/management/commands/export_members.py

```diff
@@ -104,21 +107,23 @@ def get_object_data(self, obj):
                 data["nationalities"].append(country.name)
 
         # add ordered list of addresses & coordinates
-        locations = obj.account_set.first().locations
-        if locations.exists():
-            data["addresses"] = []
-            data["coordinates"] = []
-            data["postal_codes"] = []
-            data["arrondissements"] = []
-            for location in locations.all():
-                data["addresses"].append(str(location))
-                data["coordinates"].append(
-                    "%s, %s" % (location.latitude, location.longitude)
-                    if (location.latitude and location.longitude)
-                    else ""
-                )
-                data["postal_codes"].append(location.postal_code)
-                data["arrondissements"].append(location.arrondissement() or "")
+        locations_obj = obj.account_set.first()
```

Comment:
My inclination is that we should preserve existing location info / format from previous exports for backwards compatibility so any code written against old versions will just work, but document and recommend that people use the new location export data for any new analysis.